src/Dazinator.Extensions.Options.Updatable/JsonUpdatableOptions.cs

```diff
@@ -8,68 +8,83 @@ namespace Dazinator.Extensions.Options.Updatable
     public class JsonUpdatableOptions<TOptions> : IUpdatableOptions<TOptions>
         where TOptions : class, new()
     {
-        private readonly IOptionsMonitor<TOptions> _monitor;
+        // ##############################################################################################################################
+        // Properties
+        // ##############################################################################################################################
 
-        // private IOptionsSnapshot<TOptions> _options;
-        private readonly IJsonStreamProvider<TOptions> _jsonFileStreamProvider;
-       // private readonly IOptionsMonitorCache<TOptions> _cache;
-        private readonly string _sectionName;
-        private readonly bool _leaveOpen;
-        private readonly static JsonSerializerOptions _defaultSerializerOptions = new JsonSerializerOptions() { IgnoreNullValues = true, WriteIndented = true };
+        #region Properties
 
-        public TOptions Value => _monitor.CurrentValue;
+        // ##########################################################################################
+        // Public Properties
+        // ##########################################################################################
 
-        public TOptions Get(string name) => _monitor.Get(name);
+        public TOptions Value => _Monitor.CurrentValue;
+
+        public TOptions Get(string name) => _Monitor.Get(name);
+

```

Comment:
Slight niggles, can you move the private fields to the top where they were previously, as I'#d like to avoid conflating formatting changes with fixing this particular issue if poss.

Reply:
- fyi Convention I usually apply in terms of class structure / order is:

- Static Fields and Constants
- Private fields
- Constructor/s
- Public Properties and Methods
- Dispose (if IDisposable)